VideoLocker/src/main/java/org/edx/mobile/user/DiscussionUser.java

```diff
@@ -0,0 +1,35 @@
+
+package org.edx.mobile.user;
+
+import com.google.gson.annotations.SerializedName;
+
+import java.io.Serializable;
+
+public class DiscussionUser implements Serializable {
+
+    @SerializedName("profile")
+    private Profile profile;
+
+    public Profile getProfile() {
+        return profile;
+    }
+
+    public void setProfile(Profile profile) {
+        this.profile = profile;
+    }
+
+    public static class Profile implements Serializable{
+
+        @SerializedName("image")
+        private ProfileImage image;
```

Comment:
This should probably be declared as `@Nullable`, along with it's getter and setter.